src/middlewares/__tests__/functionShield.js

```diff
@@ -0,0 +1,18 @@
+const middy = require('../../middy')
+const functionShield = require('../functionShield')
+
+describe('📦  Middleware FunctionShield', () => {
```

Comment:
can you add other tests to show that it blocks HTTP requests, spawning child processes, and accessing /tmp folder as well?

Reply:
It is not easy to do.
FunctionShield adopted to run in AWS Lambda container and requires token.
1. We should use `lambci/lambda` images instead of `circleci/node` images
2. We should put FUNCTION_SHIELD_TOKEN to circleci environment variables
3. We should find a workaround to run tests locally (probably in docker container with FUNCTION_SHIELD_TOKEN env. variable)
What do you think?

Reply:
IMO tests for HTTP request blocking and child process are out of the scope of the middleware, it should not repeat what is tested inside function shield package.

Reply:
@vladgolubev good call, at that point we're not testing the middleware anymore.